core/domain/user_services.py

```diff
@@ -551,3 +556,16 @@ def get_user_impact_score(user_id):
         return model.impact_score
     else:
         return 0
+
+def fetch_gravatar(email):
+    """Returns the gravatar corresponding to given email"""
```

Comment:
@seanlip  I can't get what exactly DEFAULT_IDENTICON_DATA_URL would store. Do you mean to store the url '/images/avatar/user_blue_72px.png' ?